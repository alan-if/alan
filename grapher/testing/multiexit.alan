The l1 Isa location
  Exit a,b,c,d,e,f To l1.
End The l1

Start At l1.
