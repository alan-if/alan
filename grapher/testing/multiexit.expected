########## multiexit ##########
digraph multiexit {
  rankdir=LR;
  node [shape=octagon;style=filled;]
  l1[label="l1"];
  l1 -> l1 [label=a];
  l1 -> l1 [label=b];
  l1 -> l1 [label=c];
  l1 -> l1 [label=d];
  l1 -> l1 [label=e];
  l1 -> l1 [label=f];
}
